Fasttext incorrect shape in test

I run all tests and get this error (any python version, `develop` branch)
CC @prakhar2b @jayantj 
```
======================================================================
FAIL: testTraining (gensim.test.test_fasttext_wrapper.TestFastText)
Test self.test_model successfully trained, parameters and weights correctly loaded
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ivan/release/test/gensim/gensim/test/test_fasttext_wrapper.py", line 63, in testTraining
    self.assertEqual(trained_model.wv.syn0.shape, (vocab_size, model_size))
AssertionError: Tuples differ: (1763, 10) != (1762, 10)

First differing element 0:
1763
1762

- (1763, 10)
?     ^

+ (1762, 10)
?     ^


----------------------------------------------------------------------
```

--- a/gensim/test/test_fasttext_wrapper.py
+++ b/gensim/test/test_fasttext_wrapper.py
@@ -54,7 +54,7 @@ class TestFastText(unittest.TestCase):
         if self.ft_path is None:
             logger.info("FT_HOME env variable not set, skipping test")
             return  # Use self.skipTest once python < 2.7 is no longer supported
-        vocab_size, model_size = 1762, 10
+        vocab_size, model_size = 1763, 10
         trained_model = fasttext.FastText.train(
             self.ft_path, self.corpus_file, size=model_size, output_file=testfile())
 

